.gitignore

```diff
@@ -1,3 +1,5 @@
+*.orig
+*.rej
```

Comment:
i would allow it, altho preferred way is that people setup their editor backup files globally ([SO post](http://stackoverflow.com/questions/7335420/global-git-ignore))